README.md

```diff
@@ -42,3 +42,22 @@ This will create an empty .modman folder in the Magento root directory. Then clo
+| bolt_boltpay_order_creation_before | global | Entry-point for applying any pre-conditions prior to the conversion of a Bolt order to Magento order | **immutableQuote**<br>_Mage_Sales_Model_Quote_<br>the Magento cart copy of the Bolt order<br><br>**parentQuote**<br>_Mage_Sales_Model_Quote_<br>the original Magento cart session<br><br>**transaction**<br>_object_<br>Bolt payload | | 
+| bolt_boltpay_order_creation_after | global | Allows for post order creation actions to be applied exclusively to Bolt orders | **order**<br>_Mage_Sales_Model_Order_<br>the converted and saved Magento order<br><br>**quote**<br>_Mage_Sales_Model_Quote_<br>the immutable quote<br><br>**transaction**<br>_object_<br>Bolt payload | | 
+| bolt_boltpay_validate_cart_session_before | global | Entry for altering the behavior of session validation prior to standard session validation | **immutableQuote**<br>_Mage_Sales_Model_Quote_<br>the Magento cart copy of the Bolt order<br><br>**parentQuote**<br>_Mage_Sales_Model_Quote_<br>the original Magento cart session<br><br>**transaction**<br>_object_<br>Bolt payload | |
+| bolt_boltpay_validate_cart_session_after | global | Entry for adding additional session validation behavior performed after standard session validation | **immutableQuote**<br>_Mage_Sales_Model_Quote_<br>the Magento cart copy of the Bolt order<br><br>**parentQuote**<br>_Mage_Sales_Model_Quote_<br>the original Magento cart session<br><br>**transaction**<br>_object_<br>Bolt payload | |
+| bolt_boltpay_validate_coupons_before | global | Entry for altering the behavior of coupon validation prior to standard coupon validation | **immutableQuote**<br>_Mage_Sales_Model_Quote_<br>the Magento cart copy of the Bolt order<br><br>**transaction**<br>_object_<br>Bolt payload | |
+| bolt_boltpay_validate_coupons_after | global | Entry for adding additional coupon validation behavior performed after standard coupon validation | **immutableQuote**<br>_Mage_Sales_Model_Quote_<br>the Magento cart copy of the Bolt order<br><br>**transaction**<br>_object_<br>Bolt payload | |
+| bolt_boltpay_validate_totals_before | global | Entry for altering the behavior of subtotal validation prior to standard subtotal validation | **immutableQuote**<br>_Mage_Sales_Model_Quote_<br>the Magento cart copy of the Bolt order<br><br>**transaction**<br>_object_<br>Bolt payload | |
+| bolt_boltpay_validate_totals_after | global | Entry for adding additional subtotal validation behavior performed after standard subtotal validation | **immutableQuote**<br>_Mage_Sales_Model_Quote_<br>the Magento cart copy of the Bolt order<br><br>**transaction**<br>_object_<br>Bolt payload | |
+| bolt_boltpay_order_received_before | global | Entry for altering the behavior of order activation and reception, (i.e. after an order payment has been authorized) | **order**<br>_Mage_Sales_Model_Order_<br>The recently converted order prior to being confirmed as authorized<br><br>**payload**<br>_object_<br>Bolt payload | |
```

Comment:
it's not clear what `order_received` means. maybe `after_authorization` reflects reality more?

Reply:
Also what is the use case of this one?